src/adapter.ts

```diff
@@ -43,9 +43,10 @@ export class TestyTsAdapter implements TestAdapter {
         fork(require.resolve('./workers/testLoader.worker.js'), [],
             { cwd: this.workspace.uri.fsPath, execPath: this.config.nodePath, execArgv: [] })
             .on('message', (response: TestSuiteInfo) => {
-                if (response instanceof String) {
+                if (response instanceof String || typeof response === "string") {
```

Comment:
The original logic does not work as you would expect see:
https://stackoverflow.com/questions/203739/why-does-instanceof-return-false-for-some-literals